src/app/providers.tsx

```diff
@@ -9,14 +9,18 @@ import { ClerkProvider } from '@clerk/nextjs'
 // reference: https://tanstack.com/query/latest/docs/framework/react/guides/advanced-ssr
 //
 import { isServer, QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { FC, ReactNode } from 'react'
 
 function makeQueryClient() {
     return new QueryClient({
         defaultOptions: {
             queries: {
                 // With SSR, we usually want to set some default staleTime
-                // above 0 to avoid refetching immediately on the client
+                // above 0 to avoid re-fetching immediately on the client
                 staleTime: 60 * 1000,
+                refetchOnWindowFocus: false,
```

Comment:
it seems like `refetchOnWindowFocus` might be beneficial?  as I understand react-query, it pauses refetch while unfocused, so the refetch on focus is good for cases where the page is in the background for extended period of time. perhaps that isn't correct though?

Reply:
yeah this is totally up for debate, there's a good article here https://tanstack.com/query/latest/docs/framework/react/guides/important-defaults I was just playing with how this would interact with page focusing and stuff - happy to remove this here and we can decide on what defaults we want (think it makes sense choosing defaults per app since different apps have different use cases)